arcgis-ios-sdk-samples/Content Display Logic/Controllers/ListViewController.swift

```diff
@@ -39,7 +39,6 @@ class ListViewController: UITableViewController {
         let cell = tableView.dequeueReusableCell(withIdentifier: "ListCell", for: indexPath)
         
         cell.textLabel?.text = list[indexPath.row]
-        cell.backgroundColor = .clear
```

Comment:
No need to explicitly set this background color - bgc for cell is default to clear color.